src/main/java/com/example/employee/exceptionHandler/EmployeeNotFoundExceptionHandler.java

```diff
@@ -0,0 +1,29 @@
+package com.example.employee.exceptionHandler;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.HandlerMapping;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@RestControllerAdvice
+public class EmployeeNotFoundExceptionHandler extends ResponseEntityExceptionHandler {
```

Comment:
We should write some test in controller tests to make sure we receive 404 when sending request to get employee which doesn't exist